cli/src/main/kotlin/commands/ScannerCommand.kt

```diff
@@ -167,5 +168,17 @@ class ScannerCommand : CliktCommand(name = "scan", help = "Run existing copyrigh
             println("Writing scan result to '$file'.")
             file.mapper().writerWithDefaultPrettyPrinter().writeValue(file, ortResult)
         }
+
+        val scanResults = ortResult.scanner?.results
+
+        if (scanResults == null) {
```

Comment:
I tried to find a code path which, searching through `scanner.scanOrtResult()` and `scanner.scanPath`, which turns this ìf-condition` to true - I couldn't find any. Could you give me a hint?

Reply:
I'm also pretty sure there is such code path that leaves `scanResults` with `null` after a scanner run. But the variable is nullable, so I need to handle the `null` case here somehow. And *if* it is `null` at this point, then certainly something did go very wrong.

Reply:
After looking into it again I believe it's fine to keep it this way.

One question popped up as I found that above mentioned functions do throw exception in case of some issues.
Do these exceptions have a similar effect as `throw ProgramResult(1)` and / or should they be caught by the comand?

Reply:
I guess they should be caught and rethrown as exceptions recognized by *clikt* to set a status code accordingly. Which exceptions did you find to be thrown?